src/components/common/ProfileArea.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function ProfileArea(props) {
+  return (
+    <div>
```

Comment:
This looks like a better way to display the info than using the ant component you were trying to use. Although some of the ant design components might still be able to be used here.